examples/README.md

```diff
@@ -0,0 +1,109 @@
+The csp-billing-adapter requires a configuration file, (defaults to `~/.config/csp_billing_adapter/config.yaml`)
+in order to run successfully. The configuration files in in the `examples` directory 
+contain examples for the different product configurations. 
+
+This README provides a listing of all of the fields and descriptions.
+Fields that are optional are labeled accordingly.
+
+```
+version:
+    The version of the adapter implementation.
```

Comment:
Should this not be "The version of the configuration structure"? The adapter version can evolve without the configuration structure changing.

Reply:
fixed in https://github.com/SUSE-Enceladus/csp-billing-adapter/pull/38